src/app/query-runner.service.ts

```diff
@@ -161,12 +160,26 @@ export class QueryRunnerService {
     query.duration = (new Date()).getTime() - query.requestSentAt.getTime();
     query.statusCode = res.status;
 
-    AppComponent.messageBarContent = {
+    AppComponent.requestStatus = {
       text: this.createTextSummary(query),
       backgroundClass: this.isSuccessful(query) ? 'ms-MessageBar--success' : 'ms-MessageBar--error',
       icon: this.isSuccessful(query) ? 'ms-Icon--Completed' : 'ms-Icon--ErrorBadge',
     };
 
+    const queries = [
+      'https://graph.microsoft.com/v1.0/me/',
+      'https://graph.microsoft.com/v1.0/user/',
+    ];
+
+    const hasQuery = queries[0].match(query.requestUrl) ||
```

Comment:
So not sure if this statement should use a loop around the `query` array to do the matching.

Reply:
Question,

Do users need to add `$select` to these specific paths ie `v1.0/me` but not to `v1.0/me/photo/$value`?